### PR TITLE
SITES-619 generate ETAGs in all show/index requests

### PR DESCRIPTION
**Pull request by freshtonic**
The generated ETAG is automatically busted when a new version of the app
is released.

NOTE: does not cover admin routes.

### CODE REVIEW

**Comment by maxious** [comment]
Categories controller?

**Comment by freshtonic** [comment]
@maxious done

**Comment by maxious** [comment]
LGTM

File: app/controllers/application_controller.rb [hunk 1]
```diff
@@ -17,12 +19,38 @@ def current_user
 
   protected
 
+  # Calls *stale?* but modifies any supplied *strong_etag* by prepending the value of 
+  # ApplicationController#etag_seed.
+  def bustable_stale?(object = nil, **kwd_args)
```
Comment (coderdan) [inline app/controllers/application_controller.rb]: I think one * is enough here, no? The double * is a c++ thing?

Comment (micapam) [inline app/controllers/application_controller.rb]: Array versus hash ... * for normal args, ** for named args

Comment (freshtonic) [inline app/controllers/application_controller.rb]: What Josh said :)

Comment (coderdan) [inline app/controllers/application_controller.rb]: Ah, right. Cool!

**Comment by coderdan** [comment]
LGTM

**Comment by freshtonic** [comment]
Waking up LGTM

**Pull request merged**